subprojects/ide/src/crossVersionTest/groovy/org/gradle/plugins/ide/tooling/m5/ToolingApiIdeaModelCrossVersionSpec.groovy

```diff
@@ -114,7 +115,8 @@ idea.module.testOutputDir = file('someTestDir')
         module.compilerOutput.outputDir == file('someDir')
         module.compilerOutput.testOutputDir == file('someTestDir')
     }
-
+    @ToolingApiVersion("<4.4")
```

Comment:
By 4.6 you mean 4.7, correct? So all the version labels in  `@ToolingApiVersion` `@TargetGradleVersion` and `@since` should reflect the 4.7 version, right?

Reply:
rebased & squashed

Reply:
Yes, I meant 4.7. Sorry for the confusion.